app/views/course/assessment/answer/_worksheet.html.slim

```diff
@@ -1,16 +1,22 @@
 - if answer.submission.attempting? && answer.specific.is_a?(Course::Assessment::Answer::Programming)
+  - programming_answer = answer.specific
+  - if answer.question.actable.attempt_limit
+    - button_text = t('.run_code_with_limit', limit: programming_answer.attempting_times_left)
+  - else
+    - button_text = t('.run_code')
```

Comment:
Actually, that is for staff to know the times left when they view or grade students' submissions, so that they will aware when students cannot run the code and their views are consistent.